src/Controllers/BatchProcessorController.ts

```diff
@@ -45,36 +60,53 @@ module StreamStats.Controllers {
         public selectedBatchProcessorTabName: string;
         public displayMessage: string;
         public isValid: boolean;
-        public AppVersion: string;  
+        public AppVersion: string;
         public submitBatchInfo: string;

```

Comment:
I don't think this variable is being used.

Reply:
Deleted!